README.md

```diff
@@ -39,17 +39,17 @@ AdministratorAccess (AWS Managed)
 - To have your DNS domain.
 
 ### Overview
-<TBD>
+
+- TBD
```

Comment:
What do you want to show here?

Reply:
I'll have AWS architecture overview here (e.g., which component will be deployed in your Aws account. etc)